controllers/api/walletRoutes.js

```diff
@@ -0,0 +1,21 @@
+const router = require('express').Router();
+const {Wallet} = require('../../models');
+
+router.get('/', async (req,res) => {
+try {
+    const userId = req.session.user_id;
+console.log(userId);
+    const wallet = await Wallet.findByPk(1);
```

Comment:
Change to findOne(where: {user_id: user.session.id})

Reply:
Got it